crates/ark-contracts/arkchain/src/orderbook.cairo

```diff
@@ -117,6 +120,8 @@ mod orderbook {
         /// Mapping of broker addresses to their whitelisted status.
         /// Represented as felt252, set to 1 if the broker is registered.
         brokers: LegacyMap<felt252, felt252>,
+        creator_brokers: LegacyMap<felt252, felt252>,
+        fulfiller_brokers: LegacyMap<felt252, felt252>,
```

Comment:
can you add a little comment there like the other part of the storage, to say it's brokerID/order_hash